run.py.txt

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python3
+
+# Generated run.py from buck2
```

Comment:
Do we want this checked into the repo?

Reply:
Nope, I created this to help repro and nvc crash without buck and it was accidentally checked in. removed.